api/bull-board/config.js

```diff
@@ -1,38 +1,24 @@
-if (process.env.NODE_ENV === 'production') {
-  const cfenv = require('cfenv'); // eslint-disable-line global-require
-  const appEnv = cfenv.getAppEnv();
+const {
```

Comment:
I can hydrate these when running the tests locally. We use the `config/test.js` convention for other parts of the app  but bull-board is a bit different since it may also be broken out into its own entity through something like the monorepo.

Reply:
yeah, i started going down a bit of a yak-shave in my head for how we should handle env vars and config...